Azure/azure-cli
# {Cognitive Services} Fix #22545: `az cognitiveservices account identity remove`: Return LROPoller

**Related command**
`az cognitiveservices account identity remove`

**Description**
Fix #22545. `client.begin_update` returns a `LROPoller` instance. During instance initialization, a thread named `LROPoller(xx)` is created. If the command's return type is poller, azure core will call `LROPoller.result()` to get the final result. Thus the `LROPoller(xx)` thread is joined and terminated.

Without `return`, azure core will not wait the thread and this thread may be still active when call `tearDown()`.


**Testing Guide**
Create a cognitive services, then
`az cognitiveservices account identity remove -g <rg> -n <service-name>`
You should see the running progress bar and the final json output.


---

This checklist is used to make sure that common guidelines for a pull request are followed.
